tests/app/execute_test.py

```diff
@@ -14,8 +14,8 @@
 
 
 @pytest.mark.gen_test
-def test_hello_world(http_client, base_url):
-    response = yield http_client.fetch(base_url)
+def test_hello_world(http_client, default_url):
```

Comment:
The jupyter server pytest-plugin we added in jupyter_server 0.2.1 drastically improves this syntax.

* You can now use `async`/`await` syntax.
* we over a `fetch` fixture—a function for making requests to the jupyter server API and extension handlers.

To see an example, check out this [simple unit test in Jupyter Server](https://github.com/jupyter/jupyter_server/blob/master/tests/services/api/test_api.py).